updatehub/src/runtime_settings.rs

```diff
@@ -87,6 +91,13 @@ impl RuntimeSettings {
         Ok(serde_json::to_string(&self.0)?)
     }
 
+    pub(crate) fn get_inactive_installation_set(&self) -> Result<Set> {
+        Ok(match self.update.upgrade_to_installation {
```

Comment:
Please add a comment on each arm explaining why we do that, this way.

Reply:
Done!